src/include/mallocMC/creationPolicies/Scatter.hpp

```diff
@@ -293,7 +293,7 @@ namespace mallocMC
                 for(;;)
                 {
                     const uint32 mask = 1 << spot;
-                    const uint32 old = alpaka::atomic::atomicOp<alpaka::atomic::op::Or>(acc, bitfield, mask);
+                    const uint32 old = alpaka::atomicOp<alpaka::AtomicOr>(acc, bitfield, mask);
```

Comment:
IIRC we have `alpaka::atomicOr(acc, bitfield, mask);` now ;)

Affects several other places as well.

Reply:
Thx for mention it. IMO no need to change. I have not inspected the changes and used a tool for all renamings.